packages/rocketchat-lib/server/models/Rooms.coffee

```diff
@@ -230,7 +230,7 @@ class ModelRooms extends RocketChat.models._Base
 	findByTypeContainingUsernames: (type, username, options) ->
 		query =
 			t: type
-			usernames: { $all: [].concat(username) }
+			usernames: { $contains: [].concat(username) }
```

Comment:
I'm wondering why this line can't just be `usernames: { $contains: [username] }`